### DISCUSSION TITLE
how to set longblob type in mysql

### DISCUSSION THREAD

**Discussion by hunjixin** [SeaQL/sea-orm#628] [General]
i use ```.binary().binary_len(1024*1024)``` set  blob, but  migrate give error ```Column length too big for column 'data' (max = 255);```

**Comment by hunjixin** [comment]
@billy1624  help

**Reply by hunjixin** [reply]
i konw how too set blob, but when insert data, got error ```Data too long for column 'data' at row 1```

**Reply by hunjixin** [reply]
seems unable to set longblob

**Comment by billy1624** [comment]
Hey @hunjixin, I'm sorry for the late reply. I think with this PR, you can now define long blob column in MySQL
- https://github.com/SeaQL/sea-orm/pull/850
